helm_tests/webserver/test_ingress_web.py

```diff
@@ -200,3 +200,27 @@ def test_can_ingress_hosts_be_templated(self):
             "cc.example.com",
             "dd.example.com",
         ] == jmespath.search("spec.rules[*].host", docs[0])
+
+    def test_backend_service_name(self):
+        docs = render_chart(
+            values={"ingress": {"web": {"enabled": True}}},
+            show_only=["templates/webserver/webserver-ingress.yaml"],
+        )
+
+        assert "release-name-webserver" == jmespath.search(
+            "spec.rules[0].http.paths[0].backend.service.name", docs[0]
+        )
+
+    def test_backend_service_name_with_fullname_override(self):
+        docs = render_chart(
```

Comment:
Sorry for confusing you.
I think for checking we don't break change here we need to apply the tests to use `Release.name` and we can achieve it by sending the function  `render_chart` in the first parameter the name of the release like that:
`render_chart('test-basic', values=..)`
and we can remove `fullnameOverride` and `useStandardNaming` from the values.

Reply:
Hi @romsharon98,

Thank you for your input!

Due to our company's design guidelines, we had to create a sub-chart from the official Airflow chart. This causes all resources to inherit the `release_name` as a prefix, which is something we want to avoid. To ensure clearer and more manageable Kubernetes resource names, we use the `fullnameOverride` parameter to customize the resource naming.

The goal of the `test_backend_service_name_with_fullname_override` is to verify that when we don't want the `release_name` as a prefix, our resources instead use the `fullnameOverride` as the desired prefix.

Hope this clarifies the approach!